deploy/sre-prometheus/ocm-agent/unsupported-logging/100-ocm-agent.PrometheusRule.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-notification-alerts
+    role: alert-rules 
+  name: elasticsearch-sre-managed-notification-alerts
```

The naming convention of all our other SRE Prometheus rules is to start with `sre-`, we should stick to that.
```suggestion
  name: sre-elasticsearch-managed-notification-alerts
```